tests/test_path.py

```diff
@@ -16,6 +16,7 @@
     find_mechanical,
     get_ansys_path,
     get_available_ansys_installations,
+    get_latest_ansys_installation,
```

Comment:
This function is in `ansys.tools.path.path` ... ;)

Feel free to add it to the `__init__` though.